internal/runners/packages/packages.go

```diff
@@ -82,6 +70,16 @@ func executePackageOperation(pj *project.Project, out output.Outputer, authentic
 
 	logging.Debug("Order changed: %v", orderChanged)
 
+	// Verify that the provided package actually exists (the vcs API doesn't care)
+	_, err = model.FetchRecipe(commitID, pj.Owner(), pj.Name(), &model.HostPlatform)
+	if err != nil {
+		rerr := &inventory_operations.ResolveRecipesBadRequest{}
+		if errors.As(err, &rerr) {
```

Comment:
Note, that this check is rather unspecific.  The `ResolveRecipesBadRequest` error could happen for other reasons.  Its payload has nothing that specifically says that an ingredient was missing: https://activestate.slack.com/archives/C7PEK0LLU/p1608058443447900

One thought that I had to work around this, is update the suggestions function in a way such that it also checks if the name and version that are provided are actually not matching any of the suggestions.  But then we still need to know if the features are case-sensitive or not, which is probably knowledge that should live in the solver.  So, maybe we need a follow-up story for pb?

Reply:
I think for this story we'd want to just make sure that you still report the error value of FetchRecipe.